app/daemon.go

```diff
@@ -131,7 +131,9 @@ func startManager(c *cli.Context) error {
 
 	m := manager.NewVolumeManager(currentNodeID, ds)
 
-	monitoring.InitMonitoringSystem(logger, currentNodeID, ds)
+	if err := monitoring.InitMonitoringSystem(logger, currentNodeID, ds, kubeconfigPath); err != nil {
```

Comment:
See my question below, are metrics best effort or are we saying they are required for longhorn operations?
This is something to add to the LEP as well

Reply:
Great point! I agree that all metrics should be best-effort. Fixed InitMonitoringSystem so that it doesn't return an error.